Then android have compile errors, for example:

  /home/runner/work/axmol/axmol/axmol/platform/android/jni/AxmolPlayerJni.cpp:326:103: error: too few arguments to function call, expected 3, have 2

--- a/axmol/base/InputSystem.h
+++ b/axmol/base/InputSystem.h
@@ -84,7 +84,7 @@ public:
     /////////////////////////////////////////////////////////////////////////////
 
     // handle key event
-    void handleKeyEvent(KeyboardEvent::KeyCode keyCode, InputPhase phase, uint32_t modifiers);
+    void handleKeyEvent(KeyboardEvent::KeyCode keyCode, InputPhase phase, uint32_t modifiers = 0);
 
     // pointer input handling helpers migrated from RenderView
     void handlePointerDown(Vec2 point, const PointerInputState& state);

